SRC/GAME.CPP

```diff
@@ -930,8 +930,10 @@ void game_shit()
     for ( a = 0; a < ACTIVE_PLAYERS; a ++  )
      aplayer[a]->move_scr();
     if ( DARK_MODE ) seeing();
-    if (ENEMIES_ON_GAME || GAME_MODE==ONE_PLAYER)
-    if ( (GAME_MODE!= NETWORK)||(NETWORK_MODE == SERVER ))
+
+    // ENEMIES_ON_GAME is an UI option visible only for deathmatch games
+    if ( ! ( ( KILLING_MODE == DEATHMATCH && !ENEMIES_ON_GAME ) ||
+        ( GAME_MODE == NETWORK && NETWORK_MODE == CLIENT ) ) )
```

Comment:
By De Morgan's laws and `(NETWORK_MODE == CLIENT) == (NETWORK_MODE != SERVER)` being true, I think this would be equal to the following. At least to me it reads much more clear than the original one :)
```
if ( ( KILLING_MODE != DEATHMATCH || ENEMIES_ON_GAME) &&
     ( GAME_MODE != NETWORK || NETWORK_MODE == SERVER) ) )
```

Reply:
Thanks, works for me!